analyticsdata/src/run_report_with_cohorts.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * Copyright 2022 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+* Google Analytics Data API sample application demonstrating the usage of
+* cohort specification in a report.
+* See https://developers.google.com/analytics/devguides/reporting/data/v1/rest/v1beta/properties/runReport#body.request_body.FIELDS.cohort_spec
+* for more information.
+* Usage:
+*   composer update
+*   php run_report_with_cohorts.php YOUR-GA4-PROPERTY-ID
+*/
+
+namespace Google\Cloud\Samples\Analytics\Data;
+
+// [START analyticsdata_run_report_with_cohorts]
+use Google\Analytics\Data\V1beta\BetaAnalyticsDataClient;
+use Google\Analytics\Data\V1beta\DateRange;
+use Google\Analytics\Data\V1beta\Dimension;
+use Google\Analytics\Data\V1beta\Metric;
+use Google\Analytics\Data\V1beta\MetricType;
+use Google\Analytics\Data\V1beta\CohortSpec;
+use Google\Analytics\Data\V1beta\CohortsRange;
+use Google\Analytics\Data\V1beta\Cohort;
+use Google\Analytics\Data\V1beta\RunReportResponse;
+
+/**
+* @param string $propertyID Your GA-4 Property ID
+* Runs a report on a cohort of users whose first session happened on the
+* same week. The number of active users and user retention rate is calculated
+* for the cohort using WEEKLY granularity.
+*/
+function run_report_with_cohorts(string $propertyId)
+{
+    // [START analyticsdata_initialize]
+    //Imports the Google Analytics Data API client library.'
```

Comment:
this isn't an import, this is instantiating the class
```suggestion
    // Create an instance of the Google Analytics Data API client library.'
```